sqlmesh/core/model/kind.py

```diff
@@ -270,6 +271,7 @@ class IncrementalByUniqueKeyKind(_Incremental):
     name: Literal[ModelKindName.INCREMENTAL_BY_UNIQUE_KEY] = ModelKindName.INCREMENTAL_BY_UNIQUE_KEY
     unique_key: SQLGlotListOfFields
     when_matched: t.Optional[exp.When] = None
+    depends_on_past: SQLGlotBool = True
```

Comment:
This preserves the existing `INCREMENTAL_BY_UNIQUE_KEY` behaviour where depends_on_past is hardcoded to `True`